components/Profile/AdminTeacherProfile/ProfileViewLeft.tsx

```diff
@@ -1,31 +1,169 @@
-import React from "react";
+import React, { useEffect, useRef, useState } from "react";
 import styles from "./ProfileViewLeft.module.css";
+import { Avatar, SxProps } from "@mui/material";
+import { Loader } from "../../util/Loader";
 
 type ProfileViewLeftProps = {
   firstName: string;
   lastName: string;
+  image: File | null;
   editProfileClicked: boolean;
   validInput: boolean;
+  imageLoading: boolean;
+  onImageChange: (img: File) => void;
   handleEditProfileClicked: () => Promise<void>;
 };
 
+type ProfilePictureProps = {
+  profileEditable: boolean;
+  onImageChange: (img: File) => void;
+  firstName: string;
+  lastName: string;
+  image: File | null;
+};
+
+const ProfilePicture: React.FC<ProfilePictureProps> = ({
```

Comment:
Maybe have some way to indicate that the pfp is clickable/editable after clicking the edit button